### ISSUE TITLE
gulp-sass not installing on windows 8,node version 10.32

### ISSUE THREAD

**Issue by nipek** [dlmanning/gulp-sass#332]
C:\Users\Ifeoluwa>npm install --save-dev gulp-sass
/

> spawn-sync@1.0.13 postinstall C:\Users\Ifeoluwa\node_modules\gulp-sass\node_mo
dules\node-sass\node_modules\sass-graph\node_modules\yargs\node_modules\os-local
e\node_modules\exec-file-sync\node_modules\spawn-sync
> node postinstall

Installing native dependencies (this may take up to a minute)
npm ERR! Failed to parse json
npm ERR! Unexpected end of input
npm ERR! File: C:\Users\Ifeoluwa\AppData\Roaming\npm-cache\uid-number\0.0.5\pack
age\package.json
npm ERR! Failed to parse package.json data.
npm ERR! package.json must be actual JSON, not just JavaScript.
npm ERR!
npm ERR! This is not a bug in npm.
npm ERR! Tell the package author to fix their package.json file. JSON.parse

npm ERR! System Windows_NT 6.2.9200
npm ERR! command "C:\\Program Files (x86)\\nodejs\\\\node.exe" "C:\\Program File
s (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "--save-dev" "gul
p-sass"
npm ERR! cwd C:\Users\Ifeoluwa
npm ERR! node -v v0.10.32
npm ERR! npm -v 1.4.28
npm ERR! file C:\Users\Ifeoluwa\AppData\Roaming\npm-cache\uid-number\0.0.5\packa
ge\package.json
npm ERR! code EJSONPARSE
npm ERR! not ok code 0

**Comment by NGalarneau** [comment]
We ran into this issue using Visual Studio 2015.  Initially we thought maybe it was the Windows issue with the path being to long do to the folder depth and tried NPM3 and flattening out dependencies like you see here http://www.ageofascent.com/2015/08/.  But didn't work, what seemed to fix the issue in Visual Studio was the version of Node in VS2015 is 10.32 and we had installed Node locally using a new version.  So inside of VS2015 we changed which version by going to "Tools -> Options -> Projects and Solutions -> External Web Tools" which shows "Locations of External Tools" then move $path below .\node_modules\.bin (should be the 2nd one).  Hope that helps
